app/controllers/DeclarationNoChangesController.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions._
+import forms.DeclarationChangesNoChangesFormProvider
+import javax.inject.Inject
+import models.Mode
+import navigation.Navigator
+import pages.{DeclarationChangesNoChangesPage, DeclarationWhatNextPage}
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.RegistrationsRepository
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import views.html.DeclarationChangesNoChangesView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DeclarationNoChangesController @Inject()(
+                                       override val messagesApi: MessagesApi,
+                                       registrationsRepository: RegistrationsRepository,
+                                       navigator: Navigator,
+                                       identify: IdentifierAction,
```

Comment:
We will need a different IdentifierAction in order to ensure they've passed IV or have the enrolment as an Agent.
I think this is impacted by @thomascurry ticket.